test/index.ts

```diff
@@ -115,3 +117,23 @@ function processEnvVars(): {
         shouldRunBenchmarks
     };
 }
+
+export const defaultParams = {
```

Comment:
should this be defaultPropertyTestParams or something to make it clear they are for prop tests?

Reply:
Fixed